### DISCUSSION TITLE
Multiple account login

### DISCUSSION THREAD

**Discussion by bohrasankalp** [community/community#4060] [Mobile] [answered]
I have multiple accounts like personal and one of company. I am expecting to Login with both accounts at a time.

**Comment by fayasnoushad** [comment]
This is a good feedback. But you can use organization.

**Reply by bohrasankalp** [reply]
Company doesn't allow to use personal mails. :)

**Comment by haldun** [comment]
Hi @bohrasankalp, thank you for the request! Please feel free to upvote [this topic that is related to multiple account support](https://github.com/github/feedback/discussions/3305).

**Comment by eliperkins** [comment, marked as answer]
Hey @bohrasankalp and folks, we've shipped functionality this week to allow you to log in with multiple accounts, like your personal and work accounts, at a the same time! [Read more about it in our Changelog post](https://github.blog/changelog/2023-08-15-log-in-with-multiple-github-accounts-on-github-mobile/).

<img width="1200" alt="image" src="https://github.com/community/community/assets/1051453/d2ba902e-2199-4625-948a-532d64a6f44e">

Download or update GitHub Mobile today from the [Apple App Store](https://apps.apple.com/us/app/github/id1477376905) or [Google Play Store](https://play.google.com/store/apps/details?id=com.github.android) to get started.
